services/ui-src/src/components/pages/ComponentInventory/ComponentInventoryPage.tsx

```diff
@@ -0,0 +1,71 @@
+import { Text, Flex, Heading } from "@chakra-ui/react";
+import { componentDefinitions, ComponentTypes } from "./component-definitions";
+import React, { ReactNode } from "react";
+
+const componentDefinition = ({
+  type,
+  description,
+  variants,
+  display,
+}: {
+  type: ComponentTypes;
+  description: string;
+  variants: ReactNode[];
+  display?: "inline" | "block";
+}) => {
+  if (variants.length === 0) return null;
+  return (
+    <Flex flexDirection="column" sx={sx.componentContainer} key={type}>
```

Comment:
Nitpick: every page should have an `h1`. Doesn't have to be dynamic like the `h2`'s.

Something like: `<h1>Component Inventory</h1>`